packages/console/src/components/Table/index.tsx

```diff
@@ -68,8 +70,8 @@ const Table = <
   const hasData = rowGroups.some(({ data }) => data?.length);
 
   return (
-    <div className={classNames(styles.container, className)}>
-      <div className={styles.tableContainer}>
+    <div className={classNames(styles.container, hasBorder && styles.hasBorder, className)}>
```

Comment:
The border should apply to the header table and the body table, otherwise, the border will display unexpectedly on some layout page.
<img width="1245" alt="image" src="https://user-images.githubusercontent.com/10806653/213222827-0b86e849-a8a9-4758-9ca4-c7808de573aa.png">

Reply:
hmmm, ok. Makes sense